CHANGELOG.md

```diff
@@ -7,8 +7,8 @@ This document contains a historical list of changes between releases. Only
 changes that impact end-user behavior are listed; changes to documentation or
 internal API changes are not present.
 
-Main (unreleased)
------------------
+## Main (unreleased)
```

Comment:
Can we leave the formatting of the h1's untouched? For reading the raw text, I find the `-------` to be easier to distinguish when scanning the file from the other `###` headers.

Reply:
oops